plugins/bars/libs/cl_bar.lua

```diff
@@ -20,7 +20,7 @@ function nut.bar.add(getValue, color, priority, identifier)
 		local oldBar = nut.bar.get(identifier)
 
 		if (oldBar) then
-			table.remove(nut.bar.list, oldBar.priority)
+			nut.bar.list[oldBar.priority] = nil
```

Comment:
This actually changes behavior because `nut.bar.list` will not act like an array.